src/Selenium2Driver.php

```diff
@@ -1211,4 +1221,21 @@ private function uploadFile($path)
         return $remotePath;
     }
 
+    /**
+     * @param $value
+     * @param $elementName
+     * @param $element
+     */
+    private function postElementValue($value, $elementName, $element)
+    {
+        $value = strval($value);
+
+        if (in_array($elementName, array('input', 'textarea'))) {
+            $existingValueLength = strlen($element->attribute('value'));
+            $value = str_repeat(Key::BACKSPACE . Key::DELETE, $existingValueLength) . $value;
+        }
```

Comment:
If I understood @stof comment (see https://github.com/minkphp/Mink/pull/767#issuecomment-431019669) correctly, then this should be moved back to `setValue` method to allow `sendKeys` to send keys (as the name suggests) instead of replacing current input value with provided one (as the `setValue` method does).